eq-author/src/App/introduction/Design/IntroductionEditor/LegalBasisField.js

```diff
@@ -1,108 +1,59 @@
 import React from "react";
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 import PropTypes from "prop-types";
 
-import { Input } from "components/Forms";
 import { colors } from "constants/theme";
-
-import iconCheck from "./icon-check.svg";
+import { Input } from "components/Forms";
 
 const LegalField = styled.div`
   display: flex;
-`;
-
-const LegalInput = styled(Input)`
-  position: absolute;
-  overflow: hidden;
-  clip: rect(0 0 0 0);
-  height: 1px;
-  width: 1px;
-  margin: -1px;
-  padding: 0;
-  border: 0;
-  &:hover,
-  &:focus {
-    border: none;
-    outline: none;
-    box-shadow: none;
-  }
+  flex-direction: column;
+  justify-content: flex-end;
 `;
 
 const LegalLabel = styled.label`
   padding: 2.5em 1.5em;
   border-radius: 4px;
+  margin-bottom: 20px;
```

Comment:
```suggestion
  margin-bottom: 1.25em;
```

Hi Sam. We usually try to use ems for sizes and spacing (e.g. padding + margins) as they're more responsive than absolute pixle values. You can use this tool to convert between pixels and ems: https://www.w3schools.com/tags/ref_pxtoemconversion.asp

(For us, 1em = 16px)

Reply:
Thank you! I've just pushed a commit to fix this.

Reply:
Also be careful because `em` changes based on the parent elements font-size